Dockerfile

```diff
@@ -1,11 +1,13 @@
 FROM python:3-alpine
 
+RUN apk add --no-cache docker=19.03.12-r0
```

Comment:
why this package?

Reply:
Legacy code. I will remove as its not needed

Reply:
ok. then please resolve when removed